codec_specs.md

```diff
@@ -872,13 +888,15 @@ VobSubs consist of two files, the .idx containing information, and the .sub, con
 The .idx file is stripped of all empty lines, of all comments and of lines beginning with `alt:` or `langidx:`.
 The line beginning with `id:` **SHOULD** be transformed into the appropriate Matroska track language element
 and is discarded. All remaining lines but the ones containing timestamps and file positions
-are put into the `CodecPrivate` element.
+are put into the `Private Data`.
 
 For each line containing the timestamp and file position data is read from the appropriate
 position in the .sub file. This data consists of a MPEG program stream which in turn
 contains SPU packets. The MPEG program stream data is discarded, and each SPU packet
 is put into one Matroska frame.
 
+Initialization: none
```

Comment:
This isn't correct; the original text states which lines must be placed in `CodecPrivate`.